bin/timeout_ssh

```diff
@@ -23,6 +23,7 @@ ARG_TIMEOUT = '--timeout='
 # program whose only job is to be a single executable that Ansible can
 # call which will then trampoline out to timeout.
 
+
```

Comment:
Nitpick: extra space here

Reply:
I am going to leave that one because my auto-formatter put that there.